stable/redis-ha/Chart.yaml

```diff
@@ -5,7 +5,7 @@ keywords:
 - redis
 - keyvalue
 - database
-version: 2.0.1
+version: 2.1.0
```

Comment:
since this chart doesn't have it, mind adding an `appVersion: 4.0.8-r0` to Chart.yaml also? That way the Redis version will show up in https://hub.kubeapps.com/charts/stable/redis-ha and other tools.

Reply:
done